bitsong/deploy.yaml

```diff
@@ -0,0 +1,47 @@
+---
+version: "2.0"
+
+services:
+  node:
+    image: ghcr.io/ovrclk/cosmos-omnibus:v0.0.10-bitsong-v0.8.0
+    env:
+      - MONIKER=node_1
+      - CHAIN_JSON=https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/chain.json
+      - CHAIN_ID=bitsong
```

Comment:
I think this CHAIN_ID is incorrect, doesn't match the chain.json. Should just be able to remove this line

Reply:
Done